lib/android/src/main/java/net/skyscanner/backpack/reactnative/flare/BpkFlareViewManager.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2020 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.reactnative.flare
+
+import android.view.View
+import androidx.annotation.VisibleForTesting
+import com.facebook.react.bridge.JSApplicationIllegalArgumentException
+import com.facebook.react.uimanager.LayoutShadowNode
+import com.facebook.react.uimanager.ThemedReactContext
+import com.facebook.react.uimanager.ViewGroupManager
+import com.facebook.react.uimanager.annotations.ReactProp
+import com.facebook.react.views.view.ReactViewGroup
+import net.skyscanner.backpack.flare.BpkFlare
+
+class BpkFlareViewManager: ViewGroupManager<RNBpkFlare>() {
+  companion object {
+    const val VIEW_NAME = "AndroidBPKFlareView"
+  }
+
+  @VisibleForTesting
+  public override fun createViewInstance(reactContext: ThemedReactContext): RNBpkFlare {
+    return RNBpkFlare(reactContext)
+  }
+
+  override fun getName(): String {
+    return VIEW_NAME
+  }
+
+  override fun addView(parent: RNBpkFlare, child: View, index: Int) {
+    parent.addViewInternal(child, index)
```

Comment:
The reason for this is because `BpkFlare` will throw an exception in case more than 1 children is added. Because of how `shadow nodes` work in react even if we wrap items in a single view in JS, it will add more than one here, which breaks the component.

To go around that we add child views into a internal wrapper, and add this wrapper to `BpkFlare`